cross/transmission/Makefile

```diff
@@ -1,26 +1,20 @@
 PKG_NAME = transmission
-PKG_VERS = 3.00
+PKG_VERS = 4.0.2
 PKG_EXT = tar.xz
 PKG_DIST_NAME = $(PKG_NAME)-$(PKG_VERS).$(PKG_EXT)
 PKG_DIST_SITE = https://github.com/transmission/transmission/releases/download/$(PKG_VERS)
 PKG_DIR = $(PKG_NAME)-$(PKG_VERS)
 
-DEPENDS = cross/zlib cross/openssl cross/curl cross/libevent
+DEPENDS = cross/curl
 
 HOMEPAGE = https://transmissionbt.com
 COMMENT  = A Fast, Easy, and Free BitTorrent Client
 LICENSE  = GPLv2/GPLv3
 
-GNU_CONFIGURE = 1
-CONFIGURE_ARGS = HAVE_CXX=yes --disable-nls --enable-daemon --enable-utp
+# gcc >= 5.1 required
+REQUIRED_MIN_DSM = 7.0
+UNSUPPORTED_ARCHS = comcerto2k
 
-ifeq ($(findstring $(ARCH), ppc824x ppc853x),$(ARCH))
-ifneq (5,$(firstword $(sort $(TCVERSION) 5)))
-# toolchains before version 5 require the old linux quota version
-# related issue: https://github.com/transmission/transmission/issues/591
-ADDITIONAL_CFLAGS = -D_LINUX_QUOTA_VERSION=1
-endif
-endif
+CMAKE_ARGS  = -DCMAKE_BUILD_TYPE=RelWithDebInfo ..
```

Comment:
This cannot just be `Release`, you actually need the deb info files as while?

Reply:
I don't quite understand what you mean. The `CMAKE_ARGS` was taken directly from [How to Build Transmission](https://github.com/transmission/transmission/blob/main/docs/Building-Transmission.md). Apologies but I'm not too familiar with `cmake`, is there a specific change you are proposing?

Reply:
Normally we build with CMAKE using `Release` type by default.  But indeed documentation here says otherwise so guessing it's all good.  Note that by it's name I'm guessing it suggest doing a build and preparing a `debian` sub-directory to be used to create a `.deb` package afterwards or from the get go, something which clearly doesn't apply when creating a Synology `.spk` package.